ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+* Provide a general summary of the issue in the **Title** above.  
+* Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+### Important
+Mark with [x] to select. Leave as [ ] to unselect.
+
+### When reporting a bug/issue, please be sure to include the following:
+- [ ] Screenshot
+- [ ] The browser, browser version and the opearting system you're using
+- [ ] The behavior you expect to see, and the actual behavior
+- [ ] Steps to reproduce the behavior 
+- [ ] Possible solution - _**not**_ obligatory, but suggest a fix/reason for the bug
+
+### When you open an issue for a change/improvement/feature request, please add as much detail as possible:
```

Comment:
This heading is pretty verbose, how about just `### When you open an issue for a change/improvement/feature request`?